module/metrics/hotstuff.go

```diff
@@ -43,6 +43,7 @@ type HotstuffCollector struct {
 	signerComputationsDuration    prometheus.Histogram
 	validatorComputationsDuration prometheus.Histogram
 	payloadProductionDuration     prometheus.Histogram
+	timeoutCollectorsRange        *prometheus.GaugeVec
```

Comment:
This is never instantiated, will cause nil pointer exception. We should create it in `NewHotstuffCollector`

Reply:
https://github.com/onflow/flow-go/pull/4311/commits/7446b996ed60c6137bccfdb72cce0e31a5a8d2ef